src/v2/spec.yaml

```diff
@@ -394,6 +394,9 @@ components:
         contract_address:
           type: string
           example: "0x325b3e2c19f4900f8f4eb5f6872dc2715e77cbac"
+        protocol_version:
```

Comment:
@MrKevinOConnell this is an optional parameter that will show up only for fabric subscriptions? what happens in paragraph's case, just ommitted?